NEWS.md

```diff
@@ -0,0 +1,107 @@
+# Changelog
```

Comment:
Sure, I will try that.

Reply:
I've added it, let me know if you prefer a different placement.

Reply:
While I initially said maybe as a last entry, I see the Misc section is ordered alphabetically, so that is a good idea to stay that way and the placement is nice :)